tests/test_storageclass_invalid.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from resources.pvc import PVC
+from tests import helpers
+from ocs import constants, defaults
+from ocs.exceptions import TimeoutExpiredError
+from ocsci.testlib import tier3, ManageTest
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier3
+class TestCaseOCS331(ManageTest):
+    def test_storageclass_cephfs_invalid(self, invalid_cephfs_storageclass):
+        """
+        Test that Persistent Volume Claim can not be created from misconfigured
+        CephFS Storage Class.
+        """
+        pvc_data = defaults.CSI_PVC_DICT.copy()
+        pvc_name = helpers.create_unique_resource_name('test', 'pvc')
+        pvc_data['metadata']['name'] = pvc_name
+        pvc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
```

Comment:
I don't think you should create any object in rook namespace (`openshift-storage` in downstream). That said, I would not block merging on this, we can fix it later via namespace fixture.